### PR TITLE
Ft contact modal

### PR DESCRIPTION
**Pull request by itolimaesther**
**What does this PR do?** 
It contains a feature of a user contacting the team.

** How should this be manually tested?** 
After cloning the repo, cd into the folder and RUN Install all necessary dependencies by running npm install Launch the node server with npm run dev Using a web browser visit the link below:** http://localhost:3500 To run local tests using Jest npm test

 **Any background context you want to provide?**
 N/A 
**What are the relevant pivotal tracker stories?** 
PT Story link - [#167590396] (https://www.pivotaltracker.com/story/show/167590396)

### CODE REVIEW

File: src/client/components/Header/index.js [hunk 1]
```diff
@@ -1,78 +1,78 @@
 import React, { useState, useContext, useEffect } from 'react';
 import { Link, NavLink } from 'react-router-dom';
 import classJoin from 'classnames';
-
-
+import { from } from 'apollo-boost';
```
Comment (devPinheiro) [inline src/client/components/Header/index.js]: Hi Esther, I see you imported Apollo-boost here. why did you import it here?

**Review by devPinheiro — CHANGES_REQUESTED** [review]

File: src/client/components/Header/index.js (hunk 1 above)
Comment (itolimaesther) [reply src/client/components/Header/index.js]: I can't remember importing that. It is not my code.

Comment (devPinheiro) [reply src/client/components/Header/index.js]: Alright, I will need you to remove that line of code since we are not using it. Thank you

**Review by devPinheiro — CHANGES_REQUESTED** [review]
Esther, you are doing a good job. Kindly smash your commits into one commit message. Kindly reach out to me if you need any help with this.🚀🚀

File: src/client/components/Header/index.js (hunk 1 above)
Comment (itolimaesther) [reply src/client/components/Header/index.js]: Done.

**Comment by itolimaesther** [comment]
> Esther, you are doing a good job. Kindly smash your commits into one commit message. Kindly reach out to me if you need any help with this.

Done

File: src/client/components/Header/index.js (hunk 1 above)
Comment (devPinheiro) [reply src/client/components/Header/index.js]: 🚀🚀

**Review by MrDiamond007 — APPROVED** [review]

File: src/client/components/Header/index.js (hunk 1 above)
Comment (MrDiamond007) [reply src/client/components/Header/index.js]: nice

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - node
+script: npm run test:coverage
```
Comment (devPinheiro) [inline .travis.yml]: Hey Esther, can you take out this line of code. We have not setup test coverage yet thanks

**Review by devPinheiro — CHANGES_REQUESTED** [review]
Kindly rebase your branch against master as there are some merge conflicts. Thank you

**Pull request closed without merging**